app/code/Magento/Review/Api/ReviewRepositoryInterface.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Review\Api;
+
+/**
+ * @api
+ */
+interface ReviewRepositoryInterface
+{
+    /**
+     * Save review.
+     *
+     * @param \Magento\Review\Api\Data\ReviewInterface $review
+     * @return \Magento\Review\Api\Data\ReviewInterface
+     * @throws \Magento\Framework\Exception\LocalizedException
+     */
+    public function save(\Magento\Review\Api\Data\ReviewInterface $review);
```

Comment:
Please split `save` into `create` and `update` because they have different validation rules. Common logic should be reused.